source/_integrations/plugwise.markdown

```diff
@@ -89,151 +89,118 @@ This integration follows standard integration removal. No extra steps are requir
 
 This will also remove all connected Adam devices (such as Anna, Tom or Lisa) or connected Adam/Stretch plugs.
 
-### Actions
+## Available actions
 
-#### Update Smile data
+The following actions, along with their examples, can be with the climate gateways:
 
-Forced update of data from your Smile can be triggered by calling the generic `homeassistant.update_entity` action with your Smile entity as the target.
+For all climate gateways:
 
-```yaml
-# Example script change the temperature
-script:
-  force_adam_update:
-    sequence:
-      - action: homeassistant.update_entity
-        target:
-          entity_id: climate.living_room
-```
-
-#### Reboot the Plugwise gateway
-
-action: `button.press`
-
-```yaml
-# Example script change the thermostat schedule
-script:
-  reboot_gateway:
-    sequence:
-      - action: button.press
-        target:
-          entity_id: button.adam_reboot
-```
-
-#### Set HVAC mode
-
-action: `climate.set_hvac_mode`
-
-Available options include `off` (Adam only) `auto`, `cool`, `heat`, and `heat_cool` (Anna with Elga only).
-
-The meaning of `off` is that the Adam regulation is set to off. This means that the connected HVAC-system does not heat or cool, only the domestic hot water heating function, when available, is active.
-
-The meaning of `cool` or `heat` is that there is no schedule active. For example, if the system is manually set to cooling- or heating-mode, the system will be active if the room temperature is above/below the thermostat setpoint.
-
-The meaning of `heat/cool` is that there is no schedule active. For example, if the system is in automatic cooling- or heating-mode, the active preset or manually set temperature is used to control the HVAC system.
-
-The meaning of `auto` is that a schedule is active and the thermostat will change presets/setpoints accordingly.
+- `climate.set_preset_mode`
+  - **Available options**: `home`, `no_frost`, `asleep` & `away`. For Anna, `vacation` is also available.
+- `climate.set_temperature`
+  - **Available setpoints**: Any within the appropriate range for the device.
+- `climate.set_hvac_mode`
+  - **Available modes**: Refer to the [HVAC modes](#hvac-modes) section below.
+- `select.select_option`
+  - **Available schedules**: Dependent on the [schedules](#schedule_management) you have configured.
```

Comment:
I did leave this in assuming #36500 would be ingested first (otherwise this is a temporary non-working link)

Reply:
Merged accordingly